packages/realm-react/package.json

```diff
@@ -12,7 +12,8 @@
     "lint": "eslint --ext .tsx ."
   },
   "dependencies": {
-    "lodash": "^4.17.21"
+    "lodash": "^4.17.21",
+    "realm": "*"
```

Comment:
Shouldn't this be set as a peer dependency now?
To signal that users need to install it alongside.
I think it would be okay if this was more specific on its version range.

Reply:
Do you need an update to the README as well?

Reply:
It is a peer dependency and I have added installing `realm` to the readme

Reply:
Oh. I'm sorry I didn't expand to see that.